tests/lib/gendeveloper1model/main.go

```diff
@@ -58,7 +57,13 @@ func main() {
 		log.Fatalf("failed to decode model headers data: %v", err)
 	}
 
-	clModel, err := devSigning.Sign(asserts.ModelType, headers, nil, "")
+	typCand, _ := headers["type"]
+	typ := asserts.ModelType
```

Comment:
nitpick: please don't abbreviate variable names, unless the abbreviation is commonly used already ("dev" for device is fine, but I have no idea what "cand" is, and if "typ" stands for "type", I find it more readable if it's written in full :-) )

Reply:
@mardy Thanks for the review, I'll update this